avocado/utils/memory.py

```diff
@@ -320,3 +320,56 @@ def get_thp_value(feature):
         return (re.search(r"\[(\w+)\]", value)).group(1)
     else:
         return value
+
+
+class MemInfo(object):
+    """
+    Representation of /proc/meminfo
+    """
+
+    class MemInfoItem(object):
```

Comment:
Sure, I don't mind about moving the inner class out. On the other hand, the DataSize class seems to me an overkill right now. Of course we can always create it in the future, if/when needed. Then we adjust the `MemInfoItem` to use it. Sending a new version. Thanks.